bin/3llo

```diff
@@ -10,10 +10,29 @@ $container.register(:api_client, Tr3llo::HTTP::Client)
 prompt = TTY::Prompt.new
 $container.register(:interface, Tr3llo::Interface.new(prompt, $stdout))
 
+def print_help(flavour)
+  Tr3llo::Presenter::HelpPresenter
+    .new($container.resolve(:interface), flavour)
+    .print!
+end
+
+unless ARGV.empty?
+  if ARGV[0] =~ /^-h|--help$/
+    print_help(Tr3llo::Presenter::FLAVOUR_HELP_CLI)
+    exit
+  end
+end
+
 configuration = Tr3llo::Configuration.new
-configuration.user_id = ENV.fetch('TRELLO_USER') { raise "Have you set TRELLO_USER?" }
-configuration.api_key = ENV.fetch('TRELLO_KEY') { raise "Have you set TRELLO_KEY?" }
-configuration.api_token = ENV.fetch('TRELLO_TOKEN') { raise "Have you set TRELLO_TOKEN?" }
+begin
+  configuration.user_id = ENV.fetch('TRELLO_USER') { raise "Have you set TRELLO_USER?" }
+  configuration.api_key = ENV.fetch('TRELLO_KEY') { raise "Have you set TRELLO_KEY?" }
+  configuration.api_token = ENV.fetch('TRELLO_TOKEN') { raise "Have you set TRELLO_TOKEN?" }
+rescue => error
+  puts "\e[31mInvalid configuration: \e[1m#{error.message}\e[0m"
```

Comment:
**ignore this**: interactive terminal program doesn't use something like https://github.com/piotrmurach/pastel anywhere for colors?

Reply:
wait 
* https://github.com/qcam/3llo/blob/838ce9d234efab421deddbe9c9a406ae5d2c4c2d/lib/core_ext.rb

looks like it